wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/EvidenceRequest.kt

```diff
@@ -706,85 +896,146 @@ fun <ResultT> EvidenceRequestIcaoView(
                     }
                     Row(
                         modifier = Modifier
-                            .fillMaxWidth()
-                            .fillMaxHeight(),
+                            .fillMaxSize()
+                            .padding(vertical = 10.dp),
                         horizontalArrangement = Arrangement.Center
                     ) {
-                        AndroidView(
-                            modifier = Modifier.padding(16.dp),
-                            factory = { context ->
-                                PreviewView(context).apply {
-                                    layoutParams = LinearLayout.LayoutParams(
-                                        ViewGroup.LayoutParams.MATCH_PARENT,
-                                        ViewGroup.LayoutParams.MATCH_PARENT
-                                    )
-                                    scaleType = PreviewView.ScaleType.FILL_START
-                                    implementationMode = PreviewView.ImplementationMode.COMPATIBLE
-                                    post {
-                                        icaoCommunication.launchCameraScan(surfaceProvider)
+                        Column {
+                            AndroidView(
+                                modifier = Modifier
+                                    .fillMaxSize()
+                                    .padding(16.dp)
+                                    .clip(RoundedCornerShape(32.dp))
+                                    .background(
+                                        MaterialTheme.colorScheme.primaryContainer.copy(alpha = 0.6f)
+                                    ),
+                                factory = { context ->
+                                    PreviewView(context).apply {
+                                        layoutParams = LinearLayout.LayoutParams(
+                                            ViewGroup.LayoutParams.MATCH_PARENT,
+                                            ViewGroup.LayoutParams.MATCH_PARENT - 100
```

Comment:
Great point!! I looked more into this (b/c afaik `MATCH_PARENT` is a constant and arithmetic operations shouldn't really work with it) and it turns out the `- 100` is ignored giving me the false sense that it was being used 🤦 It's not really needed since the UI looks exactly the same without it... I've cleaned this up - thanks!